compatibility/bazel_tools/create-daml-app/Main.hs

```diff
@@ -204,6 +204,7 @@ main = withTempDir $ \npmCache -> do
           callProcessSilent patchPath ["-p2", "-i", messagingPatch]
           forM_ ["MessageEdit", "MessageList"] $ \messageComponent ->
             assertFileExists ("ui" </> "src" </> "components" </> messageComponent <.> "tsx")
+          callProcessSilent "sed" ["-i", "s|--openssl-legacy-provider||g", "ui/package.json"]
```

Comment:
I'm confused by this, it tries to remove all mentions of ``--openssl-legacy-provider`` but `ui/package.json.template` doesn't seem to have any

Reply:
It does in this PR. That's the whole point of the PR, actually; if it weren't for our rigid setup, this PR would just be those two lines.